client/components/dangerousHtml/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+
+export default React.createClass( {
+	propTypes: {
+		tag: React.PropTypes.string.isRequired,
+		html: React.PropTypes.string.isRequired,
+		className: React.PropTypes.string,
+	},
+
+	componentDidMount() {
+		ReactDom.findDOMNode( this ).innerHTML = this.props.html;
+	},
+
+	componentDidUpdate( prevProps ) {
+		if ( prevProps.html !== this.props.html ) {
+			this.componentDidMount();
+		}
+	},
+
+	render() {
+		var tag = this.props.tag;
+		var props = {
```

Comment:
Seems to be a good case to simply pass along any inherited props, except the ones that you have custom behavior for, e.g.:

```jsx
return React.createElement( tag, omit( this.props, 'tag', 'html ) );
```

(Means you could also lose the `className` `propType`)